src/getCoinCombination.test.js

```diff
@@ -7,4 +7,59 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('1 penny = 1 cent', () => {
```

Comment:
please, describe your test well), for ex. "should return '1' pennie when '1' is entered"